### PR TITLE
Bugfix/product name root path

### PR DESCRIPTION
**Pull request by sesave**
#### What problem is this solving?

Minicart product name without /country/ on the URL. (Akamai's fault, I'm sorry).

#### How to test it?

[Workspace](https://rootpath--samsungbr.myvtex.com/!)

#### Screenshots or example usage:

image

#### How does this PR make you feel? 

hahaha

### CODE REVIEW

File: react/ProductName.tsx [hunk 1]
```diff
@@ -21,7 +22,7 @@ const ProductName: FunctionComponent = () => {
       className={`c-on-base t-title lh-copy fw6 no-underline fw5-m ${
         handles.productName
       } ${opaque(item.availability)}`}
-      href={item.detailUrl || undefined}
+      href={rootPath + item.detailUrl || undefined}
```
Comment (lucasecdb) [inline react/ProductName.tsx]: if the `item.detailUrl` doesn't exist we would end up with `<a href="undefined">` which is undesirable

```suggestion
      href={item.detailUrl ? rootPath + item.detailUrl : undefined}
```

**Review by lucasecdb — CHANGES_REQUESTED** [review]
Also add the mock for `useRuntime` so the tests don't break

File: react/ProductName.tsx (hunk 1 above)
Comment (sesave) [reply react/ProductName.tsx]: @lucasecdb Hi! Thanks for the suggestion and makes sense. It's done.

Comment (sesave) [reply react/ProductName.tsx]: @lucasecdb Can I got an example mock for useRuntime? I did one but I'm not sure that's everything ok.

File: react/__mocks__/vtex.render-runtime.tsx [hunk 2]
```diff
@@ -1,3 +1,11 @@
 import React, { FunctionComponent } from 'react'
 
 export const Loading: FunctionComponent = () => <div></div>
+
+export const useRuntime = () => {
+  let rootPath = ''
+  const setRootPath = (rP: string) => {
+    rootPath = rP
+  }
```
Comment (lucasecdb) [inline react/__mocks__/vtex.render-runtime.tsx]: we can use the other hooks here too, although you don't need to return the `setRootPath` function
```suggestion
  const [rootPath, setRootPath] = useState('')
```

Comment (sesave) [reply react/__mocks__/vtex.render-runtime.tsx]: Ready, thanks!

**Review by lucasecdb — APPROVED** [review]

**Comment by sesave** [comment]
Hi! When this it will be published? Thanks

**Comment by lucasecdb** [comment]
@vtex-apps/checkout-ui @vtex-apps/store-framework-devs can you review?

**Comment by sesave** [comment]
@lucasecdb Good morning, any update?

**Comment by lucasecdb** [comment]
> @lucasecdb Good morning, any update?

hi @sebaskun98, I'll ask my peers to review this so we can merge it. ~In the meantime, can you rebase this against master? there seems to be some conflicts~ already did 😄

**Review by jeffersontuc — APPROVED** [review]

**Review by wdsrocha — APPROVED** [review]

**Comment by sesave** [comment]
Thanks guys

**Pull request merged**